ui/dialogs/quicksearch.go

```diff
@@ -248,71 +88,21 @@ func (q *quickSearchResult) Update(result *mediaprovider.SearchResult) {
 			},
 		)
 	}
-
-	q.secondary.Refresh()
-}
-
-func (q *quickSearchResult) Tapped(_ *fyne.PointEvent) {
-	q.parent.onSelected(q.index)
-}
-
-func (q *quickSearchResult) CreateRenderer() fyne.WidgetRenderer {
-	if q.content == nil {
-		q.content = container.NewBorder(nil, nil, container.NewCenter(q.image), nil,
-			container.New(&layouts.VboxCustomPadding{ExtraPad: -15},
-				q.title,
-				q.secondary,
-			))
-	}
-	return widget.NewSimpleRenderer(q.content)
+	sr.secondary.Refresh()
 }
 
-func (q *quickSearchResult) Refresh() {
-	q.BaseWidget.Refresh()
+func (q *QuickSearch) SetOnDismiss(onDismiss func()) {
+	q.SearchDialog.OnDismiss = onDismiss
 }
 
-type quickSearchEntry struct {
-	widgets.SearchEntry
-
-	OnTypedUp     func()
-	OnTypedDown   func()
-	OnTypedEscape func()
+func (q *QuickSearch) SetOnNavigateTo(onNavigateTo func(mediaprovider.ContentType, string)) {
+	q.SearchDialog.OnNavigateTo = onNavigateTo
 }
 
-func newQuickSearchEntry() *quickSearchEntry {
-	q := &quickSearchEntry{}
-	q.ExtendBaseWidget(q)
-	q.SearchEntry.Init()
-	return q
+func (q *QuickSearch) GetMinSize() fyne.Size {
```

Comment:
Why did this have to get renamed to GetMinSize()? You can just keep the standard `MinSize` naming